applications/dashboard/controllers/class.socialcontroller.php

```diff
@@ -53,35 +53,38 @@ public function manage() {
      */
     protected function getConnections() {
         $this->fireEvent('GetConnections');
-        $Connections = [];
+        $connections = [];
 
-        foreach (Gdn::pluginManager()->AvailablePlugins() as $PluginKey => $PluginInfo) {
-            if (!array_key_exists('SocialConnect', $PluginInfo)) {
-                continue;
-            }
+        $addons = Gdn::addonManager()->lookupAllByType(\Vanilla\Addon::TYPE_ADDON);
 
-            if (!array_key_exists($PluginKey, $Connections)) {
-                $Connections[$PluginKey] = array();
-            }
+        foreach ($addons as $addonName => $addon) {
+            $addonInfo = $addon->getInfo();
 
-            $ConnectionName = $PluginInfo['Index'];
-
-            if (Gdn::addonManager()->isEnabled($PluginKey, \Vanilla\Addon::TYPE_ADDON) &&
-                (method_exists(Gdn::pluginManager()->GetPluginInstance($ConnectionName, Gdn_PluginManager::ACCESS_PLUGINNAME), 'IsConfigured'))) {
-                $Configured = Gdn::pluginManager()->GetPluginInstance($ConnectionName, Gdn_PluginManager::ACCESS_PLUGINNAME)->IsConfigured();
-            } else {
-                $Configured = null;
+            // Limit to designated social addons.
+            if (!array_key_exists('socialConnect', $addonInfo)) {
+                continue;
             }
 
-            $Connections[$PluginKey] = array_merge($Connections[$PluginKey], $PluginInfo, array(
-                'Enabled' => Gdn::addonManager()->isEnabled($PluginKey, \Vanilla\Addon::TYPE_ADDON),
-                'Configured' => $Configured
-            ), array(
-                'Icon' => sprintf("/plugins/%s/icon.png", $PluginInfo['Folder'])
-            ));
+            if (!array_key_exists($addonName, $connections)) {
```

Comment:
Probably never. I was over-cautiously reusing that check from the last version. But come to think of it there was a lot squirrelly about that version so I probably shouldn't have.